webauthn-rs/src/lib.rs

```diff
@@ -84,6 +90,13 @@ pub struct Webauthn {
     algorithms: Vec<COSEAlgorithm>,
 }
 
+impl Webauthn {
+    /// For advanced users allow direct access to the core reg/auth methods
+    pub fn advanced(&self) -> &WebauthnCore {
```

Comment:
Maybe should be "danger_core_access" or something?

Reply:
Oh, and what kind of use-cases do you forsee this for? It's a bit of a sharp edeg, I'm curious what your idea is here :)

Reply:
passwordless/securitykey methods dont seem to provide enough granularity for me, so why not open it up to those who know exactly the params they want :)

Reply:
Yeah, but then you could just use "core" directly, you don't need this wrapper then?

Reply:
I made it unsafe because using webauthn "without guide rails" is *dangerous*. There are so many security footguns and so many ways to fuck it up. I wanted anyone who "used' that library to *really* have to acknowledge that risk, and really understand what they are doing. Sadly, unsafe is the only scary keyword way to force people to actually check "why is that unsafe?".